sdks/java/maven-archetypes/examples/build.gradle

```diff
@@ -17,10 +17,53 @@
  */
 
 apply from: project(":").file("build_rules.gradle")
-applyJavaNature()
+applyJavaNature(artifactId: "beam-sdks-java-maven-archetypes-examples")
 
 description = "Apache Beam :: SDKs :: Java :: Maven Archetypes :: Examples"
 
+// Extract the version from a library:version dependency.
+def ver = { return it.substring(1+it.lastIndexOf(':')) }
```

Comment:
This doesn't work if the version has a classifier attached to it.
Please create a Dependency object and call getVersion on it:
https://docs.gradle.org/current/javadoc/org/gradle/api/artifacts/Dependency.html#getVersion--

Reply:
Done. Used dependencies { create(moduleName).getVersion() }